Source/WSTagsField.swift

```diff
@@ -372,6 +381,11 @@ open class WSTagsField: UIScrollView {
         else if self.tags.contains(tag) {
             return
         }
+        
+        var tag = tag
```

Comment:
this naming is confusing, where is the second `tag` coming from?

Reply:
Here what I wanted to do is add functionality that will allow us to modify tag before adding it, since the parameter `tag` is a `let` constant I copied into `var` to modify it. I'll change the variable name to be more descriptive of this.